ddp-workspace/projects/ddp-pancan/src/app/components/participation/participation.component.html

```diff
@@ -0,0 +1,27 @@
+<main class="participation">
+    <h2 class="title" translate>App.ParticipationPage.Title</h2>
+    <section *ngFor="let stepNumber of '12345'.split(''); let last=last;" class="step">
+        <div class="step__data">
+            <span class="step__label"
+                  [innerText]="translationStepData(stepNumber, '.Step') | translate">
```

Comment:
what do you think if we change the structure for ParticipationPage in json file? instead of the object, it might be an array. So we can replace the construction '12345'.split('') with 'App.ParticipationPage.Steps' | translate. And in general, follow the approach we chose in the faq-section component.

Reply:
Agree, it is more generic way.
Fixed at 9d6ef0c989ea9767fd5ef6d023506ef7eb753b1d